### PR TITLE
docs(langfuse): add chatlitellm section

### PR DESCRIPTION
**Pull request by udit-001**

### CODE REVIEW

File: docs/my-website/docs/langchain/langchain.md [hunk 1]
```diff
@@ -133,3 +133,6 @@ chat(messages)
 ```
 </TabItem>
 </Tabs>
+
+## Langfuse Integration
```
Comment (ishaan-jaff) [inline docs/my-website/docs/langchain/langchain.md]: Change title to -> `Use langchain ChatLiteLLM + Langfuse`

File: docs/my-website/docs/observability/langfuse_integration.md [hunk 2]
```diff
@@ -132,6 +132,40 @@ print(response)
 
 ```
 
+### ChatLiteLLM Integration
```
Comment (ishaan-jaff) [inline docs/my-website/docs/observability/langfuse_integration.md]: Change title to `Use langchain ChatLiteLLM + Langfuse`

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]

File: docs/my-website/docs/langchain/langchain.md (hunk 1 above)
Comment (udit-001) [reply docs/my-website/docs/langchain/langchain.md]: done

File: docs/my-website/docs/observability/langfuse_integration.md (hunk 2 above)
Comment (udit-001) [reply docs/my-website/docs/observability/langfuse_integration.md]: done

**Comment by ishaan-jaff** [comment]
lgtm

**Pull request merged**